toolparameteriser/utils.py

```diff
@@ -1,20 +1,23 @@
 import logging,sys,os
 
-def setlogging():
-    if not os.path.exists(f'{os.path.expanduser("~")}/.toolparametriser/'):
-        os.makedirs(f'{os.path.expanduser("~")}/.toolparametriser/')
-    FORMAT = '[%(asctime)s]|%(levelname)s|%(name)s|%(message)s'
+def setlogging(debug=False):
+    if not os.path.exists(f'{os.path.expanduser("~")}/.toolparameteriser/'):
+        os.makedirs(f'{os.path.expanduser("~")}/.toolparameteriser/')
+    FORMAT = ' %(asctime)s | %(levelname)-8s | %(name)s | %(message)s'
+
+    if debug:
+        level = logging.DEBUG
+    else:
+        level = logging.INFO
```

Comment:
A more common log level for non-debug would be `WARNING` (which is the default). Otherwise all `logging.info()` logs will be printed. Is that what you want?

Reply:
Yeah I set info which prints messaging about the progress. Would you recommend doing it differently?

Reply:
As long as that's what you intend, this is probably okay. I would say that the ideal interface would let you use `--logging debug` or `--logging info` to customize logging to a finer degree, but this is optional future work.

Reply:
Added an issue so I don't forget https://github.com/WEHI-ResearchComputing/ToolParameteriser/issues/16. Resolving thread.